src/auth/integration-tests/src/lib.rs

```diff
@@ -157,6 +159,17 @@ pub async fn workload_identity_provider_url_sourced() -> anyhow::Result<()> {
       }
     });
 
+    if with_impersonation {
+        let impersonated_email =
+            "impersonation-target@cloud-sdk-auth-test-project.iam.gserviceaccount.com";
```

Comment:
how do I run this in my test project?

Reply:
In this case I think it will be obvious. I would add "Remaining Duration should have been possible to convert to Instant but it failed" but that is easily inferrable.

Reply:
Oh running on your test project is going to take lot of setup, You could use the test project I have setup.